W 12141803 timezoneshandling se (#104)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/botmanagement-manage.adoc

Assign service times to bots to determine when a bot may execute processes.

> Service times are repeatable time intervals that define when a service is available. You assign them to bots to determine when a bot may execute processes.

Reviewer: ```suggestion
Service times are repeatable time intervals that define when a service is available. You assign them to bots to determine when a bot can execute processes.
```

Reviewer: Our Style Guide wants us to use "can" instead of "may."

> Specify the service time in one time zone. If the bot that you assign the service time to is installed on a computer that uses a different time zone, the service time interval is converted to the bot’s time zone. If you assign the service time to multiple bots in different time zones, all bots execute processes simultaneously. The time intervals might change, depending on daylight saving time.

Reviewer 2: Suggestion: If the bot you assign the service time to...

> You specify the service time in one time zone. If the bot that you assign the service time to is installed on a computer using a different time zone, the service time interval is converted to the bot’s time zone. If you assign the service time to multiple bots in different time zones, all bots will execute processes simultaneously. The time intervals may change depending on daylight saving time.

Reviewer: ```suggestion
You specify the service time in one time zone. If the bot that you assign the service time to is installed on a computer that uses a different time zone, the service time interval is converted to the bot’s time zone. If you assign the service time to multiple bots in different time zones, all bots will execute processes simultaneously. The time intervals can change depending on daylight saving time.
```

> For example, if you specify a service time starting at 9:30 AM and ending at 5:30 PM in Buenos Aires time (ART (UTC -3)) and the bot is installed on a computer in Berlin (CET (UTC +1)), the bot will execute processes between 1:30 PM and 9:30 PM in Berlin time.

Reviewer: ```suggestion
For example, if you specify a service time that starts at 9:30 AM and ends at 5:30 PM in Buenos Aires time (ART (UTC -3)) and the bot is installed on a computer in Berlin (CET (UTC +1)), the bot will execute processes between 1:30 PM and 9:30 PM in Berlin time.
```

Learn how to xref:botmanagement-support.adoc#manage-service-times[manage service times].

Assign process applications to bots that can access them.

> You can deploy a process on a bot which has assigned the required applications.

Reviewer: ```suggestion
You can deploy a process on a bot that has assigned the required applications.
```

> Down times are repeatable time intervals that define when an application is not available. Processes whose applications are currently down are executed to completion and are not restarted until the downtime period has ended.

Reviewer: ```suggestion
Down times are repeatable time intervals that define when an application is unvailable. Processes whose applications are currently down are executed to completion and are not restarted until the downtime period has ended.
```

> Down times are repeatable time intervals that define when an application is not available. Processes whose applications are currently down are executed to completion and are not restarted until the downtime period has ended.

Reviewer: Global:  Down times > Downtimes

> You specify a down time in one time zone. If the bot that you assign the down time to is installed on a computer using a different time zone, the down time interval is converted to the bot’s time zone. If you assign the application to multiple bots in different time zones, all bots will pause executing processes using that application simultaneously. The time intervals may change depending on daylight saving time.

Author: 'Specify a downtime in the time zone in which it occurs.', says DeepL....

> You specify a down time in one time zone. If the bot that you assign the down time to is installed on a computer using a different time zone, the down time interval is converted to the bot’s time zone. If you assign the application to multiple bots in different time zones, all bots will pause executing processes using that application simultaneously. The time intervals may change depending on daylight saving time.

Author: @dhirsch1

> You specify a down time in one time zone. If the bot that you assign the down time to is installed on a computer using a different time zone, the down time interval is converted to the bot’s time zone. If you assign the application to multiple bots in different time zones, all bots will pause executing processes using that application simultaneously. The time intervals may change depending on daylight saving time.

Reviewer: Sounds good

> For example, if you specify a down time starting at 1:30 AM and ending at 5:30 AM in Tokyo time (JST (UTC +9)) and the bot is installed on a computer in San Francisco (PST (UTC -8)), the bot will not execute processes between 8:30 AM and 12:30 PM in San Francisco time.

Reviewer: ```suggestion
For example, if you specify a down time that starts at 1:30 AM and ends at 5:30 AM in Tokyo time (JST (UTC +9)) and the bot is installed on a computer in San Francisco (PST (UTC -8)), the bot will not execute processes between 8:30 AM and 12:30 PM in San Francisco time.
```
